src/components/App.js

```diff
@@ -23,38 +25,35 @@ export default class App extends Component {
   }
 
   typeHandler(e, stateType){
-    if(stateType === 'venuesPerCity'){
+    stateType === 'venuesPerCity' ?
       this.setState({
         [stateType]: e
       })
-    } else {
+    :
       this.setState({
         [stateType]: e.target.value
       })
-    }
   }
 
-  capsFirstLetter(string) {
-    return string.replace(/\w\S*/g, (txt) => {return txt.charAt(0).toUpperCase() + txt.substr(1).toLowerCase();});
+  scrollAfterSearch() {
```

Comment:
loving the ternaries. Did you toss the reg expression stuff completely or was that moved into another function?